industrial_ci/scripts/run_travis

```diff
@@ -140,15 +153,22 @@ def main(scripts_dir, argv):
     args, extra_env = parse_extra_args(argv[1:])
 
     path, args = find_config_file(args, ['.travis.yml', '.travis.yaml'])
-    global_env, job_envs = read_env(read_yaml(path)['env'])
+    config = read_yaml(path)
+    global_env, job_envs = read_env(config['env'])
+    allow_failures = read_allow_failures(config)
+    job_envs =   [ x for x in job_envs if x not in allow_failures ] \
+               + [None] * read_num_include(config) \
+               + [ x for x in job_envs if x in allow_failures ]
 
     if len(args) == 0:
         if(len(global_env) > 0):
             print('Globals: %s' % str(global_env))
         jobs = len(job_envs)
         digits = len(str(jobs))
         for i in range(jobs):
-            print('Job %s: %s' % ( str(i+1).rjust(digits), job_envs[i]))
+            print('Job %s%s: %s' % ( str(i+1).rjust(digits),
+                                     ' (allow_failures)' if job_envs[i] in allow_failures else '',
+                                     job_envs[i] if job_envs[i] is not None else '<skipped include>'))
```

Comment:
I assume printing `<skipped include>` is because industrial_ci does not (yet?) support Travis CI's [include](https://docs.travis-ci.com/user/build-matrix/#explicitly-including-jobs) so skipping the job? If so not sure if this printed sentence is clear enough. Maybe "skipped 'include' job" etc.?

Reply:
Good point! I have changed it to `<unsupported job from 'include' section>`.